docs/src/styles/docs/base.scss

```diff
@@ -303,6 +303,31 @@ h4:hover > a .icon-link {
   }
 }
 
+.expand-out {
+  // Only expand things for people without reduced motion preference
+  @media (prefers-reduced-motion: no-preference) {
+    &.shown {
+      animation: expand-out calc(2 * var(--amplify-time-long))
+        cubic-bezier(0.2, 0.6, 0.3, 1) both;
```

Comment:
Out of scope for this, but should we add a longer time to our design tokens and should we add easing curves to our design tokens too?

Reply:
Addingmore animation timings makes perfect sense, for the curves, it's quite subjective and depends on the animation type. Like these particular values work well for this animation but might not for another if that makes sense.